app/ports/templates/ports/port-detail/installation_stats.html

```diff
@@ -186,6 +189,32 @@ <h4 class="modal-title">How are monthly installations calculated?</h4>
       }
     </script>
 
+
+    <!-- OS VERSIONS AND ARCHITECTURES -->
+    <script type="text/javascript">
+        google.charts.load('current', {'packages':['corechart']});
+        google.charts.setOnLoadCallback(drawChart);
+        function drawChart() {
+            {% regroup port_installations_by_stdlib_and_os_arch|dictsort:"submission__cxx_stdlib" by submission__cxx_stdlib as stdlib_chart_stdlib_parent %}
```

Comment:
We could make a backup of this commit and potentially even remove it, and implement it the way we discussed today.